source/frontend/src/models/generate/acquisition/GenerateH120InterestHolder.ts

```diff
@@ -0,0 +1,26 @@
+import { Api_InterestHolder, Api_InterestHolderProperty } from '@/models/api/InterestHolder';
+import Api_TypeCode from '@/models/api/TypeCode';
+import { formatNames } from '@/utils/personUtils';
+
+export class Api_GenerateH120InterestHolder {
+  interestHolderName: string;
```

Comment:
This file was renamed as-is from Api_GenerateInterestHolder - because this model is tightly coupled with the H120 property model